UpstreamUpdate from https://github.com/space-syndicate/space-station-14

Dushess0 opened the pull request:
Incoming upstream update from [https://github.com/space-syndicate/space-station-14](https://github.com/space-syndicate/space-station-14)  *branch=``master``*<span class='DON-NOT-MOFIY-automated-pr-status-start'/>

<details><summary class='automated-pullrequest-status'><b>Status [updated at 2022-12-13 03:02:12 UTC]</b></summary><p>

```
LastUpdateStartTimeUtc: 2022-12-13T03:02:06.1438854Z
Messages: []
Error: false
UncriticalErrors: false
NoNewCommitsOnBaseBranch: false
CreatedNewBranch: true
HasUpstreamUpdates: false
Pushed: true
CreatedPR: true
UpdatedPRSuccessfully: false
ResolvedConfig:
  IdentityEmail: action@github.com
  IdentityUsername: PullDetachedRemote 0.3.1.0
  PRMetaInfo:
    Assignees: []
    Reviewers: []
    Labels: []
  PathToWorkingRepo: /github/workspace/.git/
  CloneMode: DO_NOTHING
  OriginRepo: 
  OriginBranch: 
  UpstreamRepo: https://github.com/space-syndicate/space-station-14
  UpstreamBranch: master
  UpdateBranch: upstreamupdate/github.com/space-syndicate/space-station-14/master
  IgnoreUpstreamRepoProtocolInUpdateBranch: false
  UpstreamCredMode: AUTO
```
</p></details>
<sub>Automatically created by <a href="https://github.com/litetex/pull-detached-remote"><img src="https://raw.githubusercontent.com/litetex/PullDetachedRemote/develop/logo.png" height=15></img> PullDetachedRemote</a></sub>
<span class='DON-NOT-MOFIY-automated-pr-status-end'/>

KIBORG04 commented:
Видимо теперь бот я, так как иначе оно не запускает все воркфлоу по причине https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#triggering-a-workflow-from-a-workflow

LudwigVonChesterfield reviewed (approved):
оказывается субмодуль апдейт тест можно игнорировать иногда
"если знаешь что делаешь"

EmoGarbage404 committed:
Exosuit: Ripley (#12668)

* mechs

* interaction relay

* atmos handling

* fuck around with interaction events

SPAGHETTI CODE OH MY GOD

* more sprites and whatever the hell

* more mech shit

* more shit for equipment

* starting equipment (for nukie mechs and such)

* equipment cycling

* starting with some of the ui

* a fat chunk of ui prototyping

* done tinkering with ui

* a bunch of ui stuff and what have yous

* cleaning up grabber and state handling

* make the ui actually functional + watch me port a million icons

I swear i'll prune the sprites later blease

* start on construction

* construction yo mamma

* remove some unused files

* fix a silly

* make the graph sane

* make it actually constructible.

* print the boards as well, bozo

* rebalance part prices

* eject action

also i appease the russians by remembering to localize

* Punch Shit

* make mech integrity and repairs work

* Make the UI more based

STOMP STOMP STOMP STOMP

* make equipment even more based

* batteries and other such delights

* make the ui look pimpin af

* make the construction mega based

* UI but so epic

* equipment

* some sweat tweaks

* damage rebalancing

* restructure tech

* fix some shit

* mechs inherit access

* make icons actually use sprite specifiers

* TRAILING COMMAA!!!!!

* fix a mild indentation sin

* undo this change because it isn't needed

* actually fix this

* secret webeditting shhhh

* place this tech here

* comments

* foo

PJBot committed:
Automatic changelog update

Elijahrane committed:
blinding networking (#12937)

DrSmugleaf committed:
Fix error when spawning nuke ops (#12961)

PJBot committed:
Automatic changelog update

EmoGarbage404 committed:
fix ripley grabber bugs (#12962)

PJBot committed:
Automatic changelog update

Errant-4 committed:
Handcuffing in Harm mode (#12838)

ElectroJr committed:
Fix projectile exception (#12956)

PJBot committed:
Automatic changelog update

PaulRitter committed:
Changes needed for new notnullableflaganalyzer (#12957)

PaulRitter committed:
more proper serv4 usage because i forgot to push for the last pr (#12965)

metalgearsloth committed:
Re-save nukie planet again (#12967)

Morb0 committed:
Merge remote-tracking branch 'upstream/master' into upstream-sync

Morb0 committed:
Update locale

Morb0 committed:
Merge pull request #634 from space-syndicate/upstream-sync

Upstream sync

Morb0 committed:
Hotfix research technology icon state

Morb0 committed:
GitHub files update (#633)

lapatison committed:
Translate Upstream #634 (#635)

Co-authored-by: Morb <14136326+Morb0@users.noreply.github.com>

IanComradeBot committed:
Automatic changelog update

EmoGarbage404 committed:
mech equipment fix (#12969)

PJBot committed:
Automatic changelog update

moonheart08 committed:
Removes Pillar. (#12977)

* Removes Pillar.

I suppose this is my lesson to never trust people who claim they'll address issues later, because Pillar is a pile of issues that have been unaddressed since it's inception, despite my countless attempts (and growing frustration with) getting them handled.

There's too many issues to list here, so I'm just fully removing the map. Pillar will not be re-added without every single issue being addressed _at the time of PR_ and will be given zero leeway with leaving issues to solve later, because they clearly will never be solved.

* guh

Co-authored-by: moonheart08 <moonheart08@users.noreply.github.com>

PJBot committed:
Automatic changelog update

metalgearsloth committed:
Fix duplicate mech footstep sounds (#12972)

* Fix duplicate mech footstep sounds

We just add a new component for relay targets so we can use that for audio prediction.

* Fix cwash

* woop

PJBot committed:
Automatic changelog update

metalgearsloth committed:
Add simple miss chances for NPCs (#12978)

Doesn't consider juking potential but okay for now.

PJBot committed:
Automatic changelog update

Morb0 committed:
Fax Machine (#11704)

PJBot committed:
Automatic changelog update

ShadowCommander committed:
Update RobustToolbox (#12981)

ShadowCommander committed:
Update RobustToolbox to v0.71.1.3 (#12983)

PJBot committed:
Automatic changelog update

metalgearsloth committed:
Context steering for NPCs (#12915)

PJBot committed:
Automatic changelog update

ElectroJr committed:
Add point light to burning entities (#12959)

PJBot committed:
Automatic changelog update

metalgearsloth committed:
Replace GridEntityId with Owner (#12743)

Chief-Engineer committed:
Add stamina logs (#12921)

EmoGarbage404 committed:
Remove xenoarch boards from RD locker (#12991)

metalgearsloth committed:
Ignore non-hard bodies for npc steering (#12990)

PJBot committed:
Automatic changelog update

Cheackraze committed:
Moose 3.2.1 (#12984)

* hamter fax

* cloning swap

Cheackraze committed:
Kettle 1.4.3 (#12859)

* Kettle 1.4.3

* fax machines

* hamter

Morb0 committed:
Merge remote-tracking branch 'upstream/master' into upstream-sync

# Conflicts:
#	Resources/Prototypes/Maps/pillar.yml

Morb0 committed:
Update locale

Morb0 committed:
Merge pull request #636 from space-syndicate/upstream-sync

Upstream sync

KIBORG04 commented:
Ебать, мехи
